### ISSUE TITLE
teacherRegistration.validationAge is displayed & validation was triggered for a student

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-www#1278]
**TL;DR:** On the Month field of the teacher/ student registration demographics step, teacherRegistration.validationAge is displayed rather than the intended validation message. This validation was also triggered for a student, which I don't think it should have been.

It also seems to want me to change the month after I have triggered the warning once, even if I have changed the year to be >> 13 years ago.

**Repro:**
- Use a student sign-up link on staging
- Fill out the steps and leave the age set to its default state
- Click Next

<img width="691" alt="screen shot 2017-04-20 at 4 35 08 pm" src="https://cloud.githubusercontent.com/assets/1320456/25251526/a41eec96-25e7-11e7-94a4-64472a13c243.png">

**Comment by mewtaylor** [comment]
I just need to update in Transifex is all – will do now

**Comment by mewtaylor** [comment]
Closed in #1279
